cypress/Dockerfile

```diff
@@ -50,15 +44,44 @@ COPY ../src/tools/larft/config/ src/tools/larft/config/
 
 # Enable creation of screenshots/videos
 RUN useradd -M -d /hmda-frontend -s /bin/bash hmda_cypress_user && \
-  chown -R hmda_cypress_user:hmda_cypress_user /hmda-frontend/
+chown -R hmda_cypress_user:hmda_cypress_user /hmda-frontend/
 USER hmda_cypress_user
+# USER root
+
+# Uninstall prebuilt cypress if exists
+# RUN npm uninstall cypress -g
+# RUN rm -rf /root/.cache/Cypress/*
```

Comment:
Can we clean up the unnecessary commented out code?

Reply:
They are good future reference. Its very hard to understand cypress/yarn cache